src/app.js

```diff
@@ -0,0 +1,102 @@
+const express =require( "express");
+const dotenv =require("dotenv") ;
```

Comment:
Codacy has a fix for the issue: [Replace `require("dotenv")·` with `·require("dotenv")`](https://app.codacy.com/gh/armerillo/ql-task/pullRequest?prid=9973834)
```suggestion
const dotenv = require("dotenv");
```